### DISCUSSION TITLE
What do you do to enable repo's secrets to contributors?

### DISCUSSION THREAD

**Discussion by dianjuar** [jscutlery/semver#441] [Q&A]
Hi 👋 

I wonder how you made it possible to share the secret [`CODECOV_TOKEN`](https://github.com/jscutlery/semver/blob/main/.github/workflows/test-and-release.yml#L25) to contributors?

I made a [PR](https://github.com/jscutlery/semver/pull/381) to this repo, and the code coverage service did its work. It commented on the PR, which made me think that my PR had access to that secret.

I'm saying this because, on my repo, a first-time contributor [made a PR](https://github.com/bikecoders/ngx-deploy-npm/pull/216). On the CI configuration, a step requires access to a token placed on secrets, but the PR doesn't have access to the repo's secrets and is causing its [failure](https://github.com/bikecoders/ngx-deploy-npm/runs/5194731773?check_suite_focus=true).

The repo's secrets configuration page says:
> Secrets are environment variables that are encrypted. Anyone with collaborator access to this repository can use these secrets for Actions.
> 
> Secrets are not passed to workflows that are triggered by a pull request from a fork. [Learn more](https://docs.github.com/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets).

So I wonder, how did you make it possible?

**Comment by edbzn** [comment]
Hey @dianjuar sorry I didn't see the notification. Did you find the answer? I do nothing special to make tokens available for the forks.

**Reply by dianjuar** [reply]
I couldn't find the answer.

I gave up, I don't know how to solve that situation.
It's very frustrating, it's failing even with dependabot commits.
